### DISCUSSION TITLE
Welcome to node-red-weatherflow-udp Discussions!

### DISCUSSION THREAD

**Discussion by CrunkA3** [CrunkA3/node-red-weatherflow-udp#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by toddhutch** [comment]
I've attempted to use this in a Synology docker setup.   It doesn't appear to work, I'm us Node-RED v3.0.2 and this version.  I have a weatherflow on my local network,
3/2/2023, 4:04:20 PM[node: 26dd74b4b3c0e274](http://192.168.111.93:1880/#)
msg : error
"Error: bind EADDRNOTAVAIL 192.168.111.189:50222"
I realized after searching for this, you should use the IP address 0.0.0.0 everything works fine now. 👍
